website/server/libs/slack.js

```diff
@@ -77,14 +89,75 @@ function sendFlagNotification ({
   });
 }
 
+function sendInboxFlagNotification ({
+  authorEmail,
+  flagger,
+  message,
+  userComment,
+}) {
+  if (SKIP_FLAG_METHODS) {
+    return;
+  }
+  let titleLink;
+  let authorName;
+  let title = `Flag in ${flagger.profile.name}'s Inbox`;
+  let text = `${flagger.profile.name} (${flagger.id}; language: ${flagger.preferences.language}) flagged a PM`;
+  let footer = `<${SLACK_FLAGGING_FOOTER_LINK}?userId=${flagger.id}&chatMessageId=${message.id}|Flag this message>`;
+
+  if (userComment) {
+    text += ` and commented: ${userComment}`;
+  }
+
+  if (!message.user && message.uuid === 'system') {
+    authorName = 'System Message';
+  } else {
+    authorName = `${message.user} - ${authorEmail} - ${message.uuid}`;
+  }
+
+  let messageText = message.text;
+
+  if (flagger.id === message.uuid) {
+    messageText += `: ${flagger.profile.name} is writing to itself.`;
+  } else {
+    let sender = '';
+    let recipient = '';
+
+    if (message.sent) {
+      sender = flagger.profile.name;
+      recipient = message.user;
+    } else {
+      sender = message.user;
+      recipient = flagger.profile.name;
+    }
+
+    messageText += `: ${sender} is writing this message to ${recipient}.`;
```

Comment:
Could `sender` and `recipient` please include the email address and User ID in brackets after the name? I.e., the code should produce a message like this:
"ALittleYellowSpider (a@a.com; 3e595299-3d8a-4a10-bfe0-88f555e4aa0c) wrote this message to Alys (b@b.com; d904bd62-da08-416b-a816-ba797c9ee265)"

That will be helpful to mods in the case where two players have similar or identical names.

I think it might be good if the "sender is writing this message to recipient" message was put into the `authorName` variable, rather than into messageText. The `flagSlack.send` code below will then put that message near the top of the slack post.

In case it's helpful, this is what moderators see in slack now when a group message is flagged:

![image](https://user-images.githubusercontent.com/1495809/46278251-f62b2380-c5a8-11e8-803d-d285149aae17.png)

So if that were a private message, I'm proposing that the line that says this:
"ALittleYellowSpider - alice.harris@oldgods.net - 3e595299-3d8a-4a10-bfe0-88f555e4aa0c"
be replaced with this:
"ALittleYellowSpider (a@a.com; 3e595299-3d8a-4a10-bfe0-88f555e4aa0c) wrote this message to Alys (b@b.com; d904bd62-da08-416b-a816-ba797c9ee265)"